packages/react-components/react-rating-preview/src/components/RatingDisplay/RatingDisplay.test.tsx

```diff
@@ -1,18 +1,42 @@
 import * as React from 'react';
 import { render } from '@testing-library/react';
-//import { isConformant } from '../../testing/isConformant';
+import { isConformant } from '../../testing/isConformant';
 import { RatingDisplay } from './RatingDisplay';
 
 describe('RatingDisplay', () => {
-  // isConformant({
-  //   Component: RatingDisplay,
-  //   displayName: 'RatingDisplay',
-  // });
-
-  // TODO add more tests here, and create visual regression tests in /apps/vr-tests
-
-  it('renders a default state', () => {
-    const result = render(<RatingDisplay>Default RatingDisplay</RatingDisplay>);
-    expect(result.container).toMatchSnapshot();
+  isConformant({
+    Component: RatingDisplay,
+    displayName: 'RatingDisplay',
+    requiredProps: { count: 1160, value: 4.5 },
+  });
+  it('renders the correct number of items', () => {
+    const { getByRole } = render(<RatingDisplay max={10} />);
+    const items = getByRole('img');
+    expect(items.getElementsByClassName('fui-RatingItem').length).toEqual(10);
+  });
+  it('renders the valueText slot when a value is provided', () => {
+    const { getByRole } = render(<RatingDisplay value={3} />);
+    const items = getByRole('img');
+    expect(items.getElementsByClassName('fui-RatingDisplay__valueText').length).toEqual(1);
```

Comment:
How about `getByText`?
```suggestion
    expect(getByText('3')).toBeTruthy();
```